packages/story-editor/src/app/story/useStoryReducer/reducers/updateElementsByResourceId.ts

```diff
@@ -53,9 +51,8 @@ export const updateElementsByResourceId = (
 
   draft.pages.forEach((page) => {
     page.elements
-      .filter(
-        (element) => elementIs.media(element) && element.resource.id === id
-      )
+      .filter(elementIs.media)
+      .filter((element) => element.resource.id === id)
       .forEach((element) =>
         updateElementWithUpdater<MediaElement>(
```

Comment:
This assertion is now unnecessary because TS correctly infers the type.

Reply:
Weird that ESLint flagged this for me locally but it's not reported here on GitHub.

Reply:
Fixed in 5fee6f4add. And this over-typing was actually why I noticed that the filter wasn't working correctly TS-wise, so yeah, this should of course be changed. But weird that ESLint didn't catch. Not locally with me either actually.

Reply:
(actually fixed in 2bb37c3)